### PR TITLE
Revive #285 third person improvements

### PR DESCRIPTION
**Pull request by swamproyal**
I am reviving #285. This allows the user to rotate the camera (while in third person) by holding middle mouse button and moving the mouse. By pressing the middle mouse a single time, this toggles the "third person side view" mode; by default it is disabled. The side of the body the "third person side view" renders on can be changed with swamp_thirdperson_side = <0,1>.

The "freecam" and "third person side view" toggle can be bound to any key with +swamp_freecam bind.

### CODE REVIEW

File: lua/cinema/misc/cl_calcview.lua [hunk 1]
```diff
@@ -1,10 +1,26 @@
 ﻿-- This file is subject to copyright - contact swampservers@gmail.com for more information.
-GM.TauntCam = TauntCamera()
```
Comment (swamproyal) [inline lua/cinema/misc/cl_calcview.lua:2]: I dont know what this actually did

**Comment by swamproyal** [comment]
Bump, I've been using this for weeks and it works well

**Comment by swamproyal** [comment]
@WinterPhoenix @PYROTEKNIK can we work on this? I think it's a good feature

**Comment by PYROTEKNIK** [comment]
I'm in favor of it. I'm not really sure why my branch never got in

File: lua/cinema/misc/cl_calcview.lua (hunk 1 above)
Comment (WinterPhoenix) [reply lua/cinema/misc/cl_calcview.lua:2]: Did you even search for it? It's on Line 192

Comment (swamproyal) [reply lua/cinema/misc/cl_calcview.lua:2]: I dont remember why I removed it but yeah I put it back

**Review by WinterPhoenix — APPROVED** [review]

**Pull request merged**